### PR TITLE
java: add debug info

### PR DESCRIPTION
**Pull request by maxandersen**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/java.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `java -jar {{filename.jar}}`
 
+- Execute a .jar program with debug waiting to connect on port 5005:
```
Comment (agnivade) [inline pages/common/java.md:14]: I think this is a fairly advanced usage of the command, which unfortunately makes it out of scope for a tldr page.

Comment (maxandersen) [reply pages/common/java.md:14]: its the one thing you would need when using it from command line.

perl and gcc has info on how to do use and enable debug. 

unfortunately in java the line is just quite long and verbose; but thats why it qualifies to be in tldr as noone remembers that by heart imo.

Comment (sbrl) [reply pages/common/java.md:14]: /cc @agnivade - I think this one has slipped through the net

**Comment by maxandersen** [comment]
signed the cla - anything else I need to do ?

File: pages/common/java.md (hunk 1 above)
Comment (agnivade) [reply pages/common/java.md:14]: Well, I think I still respectfully disagree that this needs to be added. But if you think that this is fine, then I have no objections.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/java.md (hunk 1 above)
Comment (navarroaxel) [inline pages/common/java.md:14]: I think the dot in `.jar` is not well received for the linter
```suggestion
- Execute a jar file program with debug waiting to connect on port 5005: 
```

**Comment by navarroaxel** [comment]
I fixed a trialing space in the example description

**Pull request merged**